nimterop/cimport.nim

```diff
@@ -17,6 +18,11 @@ include "."/globals
 import "."/[git, paths, types]
 export types
 
+proc forceRebuild(): bool =
+  ## `-d:nimteropForceRebuild` can be temporarily put in user's config files
+  ## to guarantee rebuilds. See also #108.
+  gStateCT.nocache or compileOption("forceBuild") or defined(nimteropForceRebuild)
+
```

Comment:
Why do we need yet another way to disable caching? This doesn't really impact end users, just wrapper devs. There's already -f and cDisableCache?

Reply:
> This doesn't really impact end users, just wrapper devs

not just wrapper devs; also end users if they want to make sure their wrappers are force rebuilt (eg after an upstream update, or to debug or isolate a problem, etc)

* cDisableCache requires editing each source file you want to force-rebuild (and then you have to remember to undo each edit); not convenient.

* `-f` would only apply if you're directly compiling a wrapper, eg via `nim c -f mywrapper.nim` ; it won't work if that command is called indirectly, eg:
`nim c -f tests.nim` won't pass `-f` to `mywrapper1.nim`, `mywrapper2.nim` below:
```nim
# tests.nim:
proc runTest() = 
  doAssert "nim c -r mywrapper1.nim"
  doAssert "nim c -r mywrapper2.nim"
```

ditto with `nimble test` (there's no magical way this could pass `-f` down to the `nim c file` called by `nimble test`)

In contrast, `-d:nimteropForceRebuild` can be put (say temporarily) inside your ~/.config/nim/config.nims (or a parent dir of the things you want to force rebuild), and it'll apply robustly to every invocation.

Reply:
Well it is still in a file and perhaps far away from the wrapper in question unlike cDisableCache(). It isn't a big deal though. Can we shorten the flag name, feels quite long.